docs/source/contributing.rst

```diff
@@ -8,6 +8,8 @@ Requirements
 - `poetry <https://python-poetry.org>`_
 - `ganache <https://trufflesuite.com/ganache>`_ 7
 - `prettier-plugin-solidity <https://github.com/prettier-solidity/prettier-plugin-solidity>`_
+- `jq <https://stedolan.github.io/jq/>`
+- `realpath <https://www.gnu.org/software/coreutils/manual/html_node/realpath-invocation.html#realpath-invocation>` 
```

Comment:
I think we should mention `realpath` only for macOS case since otherwise it's pretty much there.
And I remember that @GabrielBuragev said that can be installed via `coreutils`:
```
brew install coreutils
```
(please correct me if I'm wrong).

So instead of mentioning `realpath`, I would just mention to install `coreutils` if on mac.

Reply:
Exactly @istankovic.

This needs to be installed: https://formulae.brew.sh/formula/coreutils